src/styles/card.scss

```diff
@@ -0,0 +1,78 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  transition-duration: 300ms;
```

Comment:
The transition property is used without specifying which properties the transition effect should apply to. According to the checklist, you should specify transition properties instead of using the shorthand for all properties to avoid unnecessary performance overhead.